.github/workflows/dependency-review.yml

```
@@ -11,4 +11,4 @@ jobs:
       - name: 'Checkout Repository'
         uses: actions/checkout@v4
       - name: Dependency Review
-        uses: actions/dependency-review-action@main
+        uses: StacklokLabs/trusty-dependency-review-action@add_trusty_scores
```

```suggestion
        uses: actions/dependency-review-action@main
```

please do not touch this workflow file